### ISSUE TITLE
Autocomplete not available when typing 

### ISSUE THREAD

**Issue by Ezwen** [element-hq/element-x-android#2139]
### Steps to reproduce

- I'm in a room screen
- I click on the text input field
- i start typing a word

#### What did you expect?

The small autocomplete area should appear above the virtual keyboard, as it does in other apps.

#### What happened instead?

No autocomplete is possible.

### Your phone model

Xiaomi Mi Mix 2

### Operating system version

lineage microg 20-20231204-microG-chiron

### Application version and app store

Element X 0.4.0

### Homeserver

Personnal private one

### Will you send logs?

Yes

### Are you willing to provide a PR?

No

**Comment by jmartinesp** [comment]
Interesting, autocomplete is enabled in the `EditText` component and is working fine for me with several keyboards. Which keyboard are you using, so I can try to reproduce the issue?

**Comment by Ezwen** [comment]
I am using the AOSP keyboard (`com.android.inputmethod.latin`), version 13, installed as system app in Lineage 20 (hence Android 13).

I'll try with another keyboard just to check!

**Comment by Ezwen** [comment]
Indeed, I just tried with [Openboard](https://github.com/openboard-team/openboard) and autocomplete is then available in Element X. 

Strange! And the strangest is that my aosp keyboard does provide autocomplete in other apps (eg classic Element).

**Comment by Cadair** [comment]
I can also confirm this issue with the AOSP keyboard.

**Comment by SeineEloquenz** [comment]
I can confirm this issue still occurring with AOSP keyboard on Lineageos 21 on a OnePlus 8T

**Comment by SeineEloquenz** [comment]
Since the update to  `v0.4.13` I no longer notice this problem.
